tests/testthat/tests_estDailyFromSurfaces.R

```diff
@@ -0,0 +1,28 @@
+context("testing estDailyFromSurfaces")
+
+test_that("estDailyFromSurfaces returns expected values", {
+  
+  skip_on_cran()
+  
+  # Uses original and "stale" versions of Choptank data created in `tests/helper-originaldata.R`
+  
+  # everything but daily has been modeled/altered in modelEstimation
+  eList_Ch_estsurf <- mergeReport(info_stale_Ch, daily_orig_Ch, sample_stale_Ch, surfaces_stale_Ch)
+  daily_est <- estDailyFromSurfaces(eList_Ch_estsurf)
+  
+  # estDailyFromSurfaces adds three columns to Sample
+  new_daily_cols <- setdiff(names(daily_est), names(daily_orig_Ch))
+  expect_equal(sort(new_daily_cols), sort(c("yHat", "SE", "ConcDay", "FluxDay", "FNConc", "FNFlux")))
+  
+  # test that no original columns were lost in estCrossVal
+  expect_true(all(names(daily_orig_Ch) %in% names(daily_est)))
+  
+  # verify that values of yHat, SE, and ConcHat are what they should be
```

Comment:
update list from yHat, SE, and ConcHat to c("yHat", "SE", "ConcDay", "FluxDay", "FNConc", "FNFlux"). or change to "new columns"